02-dependencies.md

```diff
@@ -134,6 +137,19 @@ Setting this to `true` will disable the convention build and have tipi rely on t
 
 > Note: this property expect a `boolean` value, so the deps specification shall not contain quotation marks 
 
+#### - `opts` : defines and compile-time options
+
+Enables setting compile time definitions and options in the project/dependecy context. Theses `opts` have to contain valid CMake Syntax. For example to pass _#defines_ or compile options this way simply add:
+
+```json
+{
+  (...snip...),
+  "opts": "add_compile_options( -fmath-errno -Wextra )\r\nadd_compile_definitions( DEFINE_TO_PASS_WITHOUT_D_BEFORE=1 )"
```

Comment:
I don't know what  -fmath-errno does but it looks like abi-breaking, and should not be used in such an example that is for isolated project non-abi-modifying options.